miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -58,11 +58,6 @@ proc.authenticate_account_origin
     # assert that the caller is from the user context
     exec.account::authenticate_procedure
     # => [storage_offset, ...]
-
-    # TODO: use the storage_offset for storage access
-    # drop the storage_offset
-    drop
-    # => [...]
```

Comment:
We should update the comments for this procedure to indicate that we now return `storage_offset`.